topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,159 @@
+
+## Get data
+
+### Data Upload
+
+> <hands-on-title> Getting datasets </hands-on-title>
+> 1. Create a new history for this tutorial.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the datasets from [Zenodo]({{ page.zenodo_link }})  into your Galaxy instance:
+>    ```
+>    https://zenodo.org/records/14216972/files/transcriptome.tsv
+>    https://zenodo.org/records/14216972/files/proteome.tsv
+>    https://zenodo.org/records/14216972/files/metabolome.tsv
+>    ```
+{: .hands_on}
+
+
+# Running MultiGSEA
+
+In this step, you'll use the MultiGSEA tool to perform GSEA-based pathway enrichment on the uploaded datasets.
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. Run {% tool [multiGSEA](toolshed.g2.bx.psu.edu/repos/iuc/multigsea/multigsea/1.12.0+galaxy0) %} with the following parameters
+>    - *"Select transcriptomics data"*: `Enabled`
+>        - {% icon param-file %} *"Transcriptomics data"*: `Transcriptomics`
+>        - {% icon param-select %} *"Gene ID format in transcriptomics data"*: `SYMBOL`
+>    - *"Select proteomics data"*: `Enabled`
+>        - {% icon param-file %} *"Proteomics data"*: `Proteomics`
+>        - {% icon param-select %} *"Gene ID format in proteomics data"*: `SYMBOL`
+>    - *"Select metabolomics data"*: `Enabled`
+>        - {% icon param-file %} *"Metabolomics data"*: `Metabolomics`
+>        - {% icon param-select %} *"Metabolite ID format"*: `HMDB`
+>    - *"Supported organisms"*: `Homo sapiens (Human)`.
+>    - *"Pathway databases"*: `Kegg`, `Reactome`
+>    - *"Combine p-values method"*: `Stouffer`
+>    - *"P-value correction method"*: `BH`
+>
+>    > <tip-title>About the parameters</tip-title>
+>    > - **Pathway databases**: Databases often contain their own format in which pathway definitions are provided. So you can select a relevant database. For the tutorial we choose the preset "Kegg" and "Reactome".
+>    > - **Combine p-values method**: Choose a method (here `Stouffer` for balanced weighting). To more comprehensively measure a pathway response, multiGSEA provides different approaches to compute an aggregated p value over multiple omics layers. Because no single approach for aggregating p values performs best under all circumstances, Loughin proposed basic recommendations on which method to use depending on structure and expectation of the problem. If small p values should be emphasized, Fisher’s method should be chosen. In cases where p values should be treated equally, Stouffer’s method is preferable. If large p values should be emphasized, the user should select Edgington’s method. Figure 2 indicates the difference between those three methods.
```

Comment:
```suggestion
>    > - **Combine p-values method**: Choose a method (here `Stouffer` for balanced weighting). To more comprehensively measure a pathway response, multiGSEA provides different approaches to compute an aggregated p value over multiple omics layers. Because no single approach for aggregating p values performs best under all circumstances, {% cite LOUGHIN2004467 %} proposed basic recommendations on which method to use depending on structure and expectation of the problem. If small p values should be emphasized, Fisher’s method should be chosen. In cases where p values should be treated equally, Stouffer’s method is preferable. If large p values should be emphasized, the user should select Edgington’s method. Figure 2 indicates the difference between those three methods.
```